apps/frontend/src/pages/leaderboard/[chipIssuer].tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { toast } from "sonner";
+import { CursiveLogo } from "@/components/ui/HeaderCover";
+import {
+  LeaderboardPosition,
+  LeaderboardEntries,
+  ChipIssuer,
+  ChipIssuerSchema,
+} from "@types";
+import {getTopLeaderboardEntries, getUserLeaderboardPosition} from "@/lib/chip";
+
+const communitiesEnum : { [key: string]: string } = {
+  "edge_lanna": ChipIssuer.EDGE_CITY_LANNA,
+  "devcon": ChipIssuer.DEVCON_2024,
+  "testing": ChipIssuer.TESTING,
+};
+
+const communitiesHumanReadable : { [key: string]: string } = {
+  "edge_lanna": "Edge City Lanna",
+  "devcon": "Dev Con 2014",
+  "testing": "Testing",
+};
```

Comment:
Should these be moved to constants or can they live here?

Reply:
we can keep them here until they start getting used in many more places